packages/discord.js/src/structures/GuildMember.js

```diff
@@ -84,6 +84,17 @@ class GuildMember extends Base {
     } else if (typeof this.avatar !== 'string') {
       this.avatar = null;
     }
+
+    if ('banner' in data) {
+      /**
+       * The guild member's banner hash.
+       * @type {?string}
+       */
+      this.banner = data.banner;
+    } else {
+      this.banner ??= null;
```

Comment:
Hi, is this a mistype or this operator actually works ? 

If it's not a mistype, could you explain me that operator (??=, i know that ?? is null coalescence but i'd never saw ??= before)

Thanks !

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment
please use your favourite search engine for questions, instead of code reviews

Reply:
It's not a mistype, you can learn about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment)